docs/configuration.md

```diff
@@ -2399,47 +2399,15 @@ the driver or executor, or, in the absence of that value, the number of cores av
 Please refer to the [Security](security.html) page for available options on how to secure different
 Spark subsystems.
 
-### Spark SQL
-
-Running the <code>SET -v</code> command will show the entire list of the SQL configuration.
-
-<div class="codetabs">
-<div data-lang="scala"  markdown="1">
 
-{% highlight scala %}
-// spark is an existing SparkSession
-spark.sql("SET -v").show(numRows = 200, truncate = false)
-{% endhighlight %}
-
-</div>
-
-<div data-lang="java"  markdown="1">
-
-{% highlight java %}
-// spark is an existing SparkSession
-spark.sql("SET -v").show(200, false);
-{% endhighlight %}
-</div>
-
-<div data-lang="python"  markdown="1">
-
-{% highlight python %}
-# spark is an existing SparkSession
-spark.sql("SET -v").show(n=200, truncate=False)
-{% endhighlight %}
-
-</div>
-
-<div data-lang="r"  markdown="1">
-
-{% highlight r %}
-sparkR.session()
-properties <- sql("SET -v")
-showDF(properties, numRows = 200, truncate = FALSE)
-{% endhighlight %}
+{% for static_file in site.static_files %}
+    {% if static_file.name == 'sql-configs.html' %}
```

Comment:
Hm, @nchammas did it work in your local just for doubly sure? It didn't work in my local... static_files seems only referring the files under `_site`.

Reply:
Yup, I've tested everything out. What version of Jekyll are you running?

Here's what I've got:

```
gem install jekyll:4.0.0 jekyll-redirect-from:0.16.0 rouge:3.15.0
pip install sphinx==2.3.1 mkdocs==1.0.4 numpy==1.18.1
```

By the way, I am planning to address the fact that we don't pin our doc build dependencies in a follow-up PR.

Reply:
Okay .. seems I tested wrongly.. it works in my local too.